pkg/pipeline/remote/remote.go

```diff
@@ -0,0 +1,23 @@
+package remote
+
+import (
+	"errors"
+	"github.com/rancher/rancher/pkg/pipeline/remote/github"
+	"github.com/rancher/rancher/pkg/pipeline/remote/gitlab"
+	"github.com/rancher/rancher/pkg/pipeline/remote/model"
+	"github.com/rancher/types/apis/project.cattle.io/v3"
+)
+
+func New(config interface{}) (model.Remote, error) {
+	if config == nil {
+		return github.New(nil)
```

Comment:
should we return error?

Reply:
When no config is used we use default github config for our example repos, so we return this here